components/api-gateway-migrator/pkg/migrator/translate.go

```diff
@@ -0,0 +1,144 @@
+package migrator
+
+import (
+	"encoding/json"
+	"fmt"
+
+	gatewayv1alpha1 "github.com/kyma-incubator/api-gateway/api/v1alpha1"
+	oldapi "github.com/kyma-project/kyma/components/api-controller/pkg/apis/gateway.kyma-project.io/v1alpha2"
+	rulev1alpha1 "github.com/ory/oathkeeper-maester/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func translateToApiRule(oldApi *oldapi.Api) (*gatewayv1alpha1.APIRule, error) {
+
+	newApi := gatewayv1alpha1.APIRule{}
+	newApi.Name = oldApi.Name
+	newApi.Kind = "APIRule"
+	newApi.APIVersion = "gateway.kyma-project.io/v1alpha1"
+	newApi.Namespace = oldApi.Namespace
+
+	newApi.Spec = gatewayv1alpha1.APIRuleSpec{}
+
+	port := uint32(oldApi.Spec.Service.Port)
+	isExternal := false
+
+	newApi.Spec.Service = &gatewayv1alpha1.Service{
+		Name:       &oldApi.Spec.Service.Name,
+		Port:       &port,
+		Host:       &oldApi.Spec.Hostname,
+		IsExternal: &isExternal,
+	}
+
+	gateway := "kyma-gateway.kyma-system.svc.cluster.local" //TODO: Configurable?
```

Comment:
Imho gateway should be an input parameter with `kyma-gateway` set as the default value

Reply:
done.